modules/ROOT/pages/_partials/include-permissions-api-manager-post-crowd.adoc

```diff
@@ -0,0 +1,11 @@
+API Manager Environment Administrator:: Enables you to view, create, modify, and delete APIs in the specified environment. You can also execute any actions related to API configurations, groups, proxies, alerts, contracts, tiers, policies, and other settings in the specified API Manager environment.
```

Comment:
I would add something related to automated policies. This roles enables the user to manage automated policies

Reply:
```suggestion
API Manager Environment Administrator:: Enables you to view, create, modify, and delete APIs in the specified environment. You can also execute any actions related to API configurations, groups, proxies, alerts, contracts, tiers, policies, automated policies, and other settings in the specified API Manager environment.
```